qiskit/transpiler/target.py

```diff
@@ -817,7 +817,7 @@ def check_obj_params(parameters, obj):
             if parameters is not None:
                 obj = self._gate_name_map[operation_name]
                 if inspect.isclass(obj):
-                    # The parameters argument was set and the operation_name specified is
+                    # The parameters' argument was set and the operation_name specified is
```

Comment:
The apostrophe here isn't correct - it should either be as it was before, or "The 'parameters' argument" (though also, this is just a code comment).

Reply:
Undone in 8c4650414d3dac6c960948e648458f0d495dfd4a. Thanks!